core_lib/src/tool/stroketool.cpp

```diff
@@ -208,10 +208,14 @@ bool StrokeTool::event(QEvent *event)
     if (event->type() == QEvent::Leave && !isActive()) {
         mCanvasCursorEnabled = false;
         updateCanvasCursor();
+        QObject::event(event);
+        return true;
```

Comment:
Maybe it was a bad idea to delegate event with a return value 🤔 
Shouldn't we just call event->accept(), so the event knows that it should be accepted?

also I can't find any resources that say that you should call QObject::event explicitly, since we already have that in the parent widget?